installers/redhat-docker/radiasoft-download.sh

```diff
@@ -25,23 +25,13 @@ umount /var/lib/docker
 rmdir /var/lib/docker
 perl -pi -e "s{^/var/lib/docker.*}{}" /etc/fstab
 
-Then re-run this command
+Then re-run this command. This should only happen in development environments.
```

Comment:
Maybe this command should be called redhat-docker-dev? The comment on line 33 is a dev-only comment. This can't be run with rsconf.

Reply:
Hmm why can't this be run with rsconf? That is exactly what I'm trying to enable. We currently have this code roughly copied in rsconf https://github.com/radiasoft/rsconf/pull/522/files . So, I'm trying to eliminate that duplication.

Reply:
Ok. The parameters bug me. I'm going to see if I can split out the installer into setup and install. That way rsconf can just call the install part and manage the rest of the setup (ex selinux) itself. 

If that doesn't work I'll use params.